### DISCUSSION TITLE
How to format selected filters and text results in search results

### DISCUSSION THREAD

**Discussion by egwrk** [microsoft-search/pnp-modern-search-layouts#5] [Q&A]
I have the "Show selected filters" option enabled in Search Results. However, the default format does not seem to be very user-friendly. Is there a way to change the format so that it has the desired layout below? I'm also trying to use CSS to do the below. Please refer to the image for a visual of the desired layout. 

- Have selected filters display as chips 
- Button to clear all filters
- If the filter is selected, or results match the search term, bold the matching text. Example: 'Cyber Security' filter is selected which makes 'Cyber Security' bolded text in the results. 
- There is ellipses to break up long text

For the ellipses, I tried the CSS below but it just cuts off the text instead of using an ellipses. 

```
    .template--listItem--tags {
        white-space: nowrap;
        width: 70%;
        overflow: hidden;
        text-overflow: ellipsis;
    }
```

I also tired searching through the templates here: https://github.com/microsoft-search/pnp-modern-search-layouts/tree/main/Results
However, there are not many options and none seem to have what I'm looking for.

![PNP-Layout](https://github.com/microsoft-search/pnp-modern-search-layouts/assets/161059375/db19a5d3-72d8-429d-9896-7f5014c2d8be)

**Comment by kasperbolarsen** [comment]
I would recommend looking at the guidance at https://microsoft-search.github.io/pnp-modern-search/#extensibility-model 
To me it should like a new version of the Refiner web part might be the way to go, and we would to see a PR from you :-)
